### ISSUE TITLE
Problem with App

### ISSUE THREAD

**Issue by Peppe-73** [rivafarabi/deckboard#1088]
You need to update the Android App on your tablet or you will find this display bug, photo number 1 and the App as seen from the photo are visible in the code buttons that were not seen before, the version and 2.1.1

photo number 2 is the software in the PC and it can be seen correctly, the App should be seen as in photo 2

![App](https://github.com/rivafarabi/deckboard/assets/50875260/c87d9d33-8a82-4bb5-8e5b-10127f77e399)
![PC](https://github.com/rivafarabi/deckboard/assets/50875260/0b8e1660-2331-4f49-97b8-7f1adecce750)

**Comment by schousda** [comment]
Do you use the Beta-Channel on your tablet?
As far as i can see, you are already using the v3 Software. This will not work without the beta on your device.

**Comment by Peppe-73** [comment]
On the tablet, as already written, I am using version 2.1.1, no beta version, there have been no updates to the App for more than a year, can you update the App so that it is compatible with v3?

**Comment by schousda** [comment]
In the Play Store, you need to activate the Beta program. Just scroll down in the store page of the app and join the beta program. Then you will be able to install the version 3 app.

**Comment by Peppe-73** [comment]
Hi, nothing happens, I tap on the participate item of the beta program and no window opens, I remain stuck in the App window

**Comment by Peppe-73** [comment]
Ok, I managed to get the beta, the problems reported above have been resolved, but as you can see from the screenshot the number is not centered

![WhatsApp Image 2024-05-12 at 21 13 28](https://github.com/rivafarabi/deckboard/assets/50875260/77824c0e-ad08-4e85-b636-4f3e5afd056b)

**Comment by rivafarabi** [comment]
Hmm.. the Followers graph does not show either?

**Comment by Peppe-73** [comment]
No, in fact the graphic that is in the PC version cannot be seen on the App
